components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/service/APIKeyValidationService.java

```diff
@@ -539,6 +539,24 @@ public boolean validateScopes(TokenValidationContext tokenValidationContext, Str
         return keyValidationHandler.validateScopes(tokenValidationContext);
     }
 
+    /**
+     * Validates the subscriptions of a particular API.
+     *
+     * @param context     Requested context
+     * @param version Version of the API
+     * @param appId Application ID

```

Comment:
Let's add tenant domain and exception details to java docs

Reply:
resolved, removed unused exceptions throws